Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -390,6 +418,10 @@ void ImageEditor::keydown_event(GUI::KeyEvent& event)
 
 void ImageEditor::keyup_event(GUI::KeyEvent& event)
 {
+    if (event.key() == Key_Alt) {
+        m_select_color_mode = false;
+    }
```

Comment:
This is also no longer needed now right?

Reply:
Slight oversight on my end. Nice Catch! Should be fixed now